pycrires/pipeline.py

```diff
@@ -3726,108 +3726,90 @@ def correct_wavelengths(self, nod_ab: str = "A") -> None:
         print(f"Reading telluric model spectrum...", end="", flush=True)
 
         transm_spec = np.loadtxt(self.calib_folder / "run_skycalc/transm_spec.dat")
+        
+        transm_interp = interpolate.interp1d(
+            transm_spec[:, 0], transm_spec[:, 1], kind="linear", bounds_error=True
+        )
 
         print(" [DONE]")
 
         # Read extracted spectra
 
-        fits_file = f"{self.path}/product/cr2res_obs_nodding_extracted{nod_ab}.fits"
-
-        print(f"Reading spectra of nod {nod_ab}...", end="", flush=True)
-
-        hdu_list = fits.open(fits_file)
-
-        print(" [DONE]")
-
-        # Objective function for maximizing the correlation
-        # with the telluric model spectrum from SkyCalc
-
-        def corr_func(param, *args):
-            obs_spec, model_interp = args[0], args[1]
-            wavel_new = param[0] + param[1] * obs_spec[:, 0]  # (nm)
+        fits_files = list(self.file_dict[f'OBS_NODDING_EXTRACT{nod_ab}'].keys())
+        #print(files)
 
-            try:
-                model_spec = model_interp(wavel_new)
-                corr_value = -1.0 * np.sum(
-                    obs_spec[:, 1] / np.median(obs_spec[:, 1]) * model_spec
-                )
-
-            except ValueError:
-                corr_value = np.inf
-
-            # method, times = signal.choose_conv_method(
-            #     obs_spec[:, 1], model_spec, mode="valid", measure=True
-            # )
-
-            # corr = signal.correlate(
-            #     obs_spec[:, 1] / np.median(obs_spec[:, 1]),
-            #     model_spec / np.median(model_spec),
-            #     mode="valid",
-            #     method="direct",
-            # )
-
-            return corr_value
-
-        transm_interp = interpolate.interp1d(
-            transm_spec[:, 0], transm_spec[:, 1], kind="linear", bounds_error=True
-        )
+        #fits_file = f"{self.path}/product/cr2res_obs_nodding_extracted{nod_ab}_000.fits"
```

Comment:
These commented lines can also be removed

Reply:
Done